### ISSUE TITLE
How can i fine-tune model?

### ISSUE THREAD

**Issue by TemaDobryyR** [OhadRubin/SmBop#38]
I try to fine-tune SmBop on my own small dataset. I add my sqlite file in database, update *table.json* file and create *config.json* like this
```
{
    "dataset_reader": {
        "type": "smbop",
        "dataset_path": "dataset/database",
        "keep_if_unparsable": false,
        "lazy": false,
        "limit_instances": -1,
        "max_instances": 1000000,
        "question_token_indexers": {
            "tokens": {
                "type": "pretrained_transformer",
                "model_name": "Salesforce/grappa_large_jnt"
            }
        },
        "tables_file": "dataset/output_new_table.json",
        "value_pred": true
    },
     "validation_dataset_reader": {
        "type": "smbop",
        "dataset_path": "dataset/database",
        "keep_if_unparsable": true,
        "lazy": false,
        "limit_instances": -1,
        "max_instances": 1000000,
        "question_token_indexers": {
            "tokens": {
                "type": "pretrained_transformer",
                "model_name": "Salesforce/grappa_large_jnt"
            }
        },
        "tables_file": "dataset/output_new_table.json",
        "value_pred": true
    },
    "train_data_path": "dataset/fine_train.json",
    "validation_data_path": "dataset/fine_test.json",
    "model": {
        "type": "from_archive",
        "archive_file": "model.tar.gz"
    },
    "data_loader": {
        "batch_sampler": {
            "type": "bucket",
            "batch_size": 8,
            "sorting_keys": [
                "enc",
                "depth"
            ]
        }
    },
    "validation_data_loader": {
        "batch_size": 8,
        "shuffle": true
    },
    "trainer": {
        "num_epochs": 10,
        "optimizer": {
            "type": "adam",
            "lr": 1.86e-06,
            "parameter_groups": [
                [
                    [
                        "question_embedder"
                    ],
                    {
                        "lr": 3e-08
                    }
                ]
            ]
        },
        "learning_rate_scheduler": {
            "type": "polynomial_decay",
            "power": 0.5,
            "warmup_steps": 1
        },
        "num_gradient_accumulation_steps": 4,
        "grad_norm": null,
        "patience": 100,
        "checkpointer": {
            "num_serialized_models_to_keep": 1
        },
        "grad_clipping": null,
        "validation_metric": "+spider",
        "use_amp": true,
        "cuda_device": 0
    }
}
```
and than run *resume.py* with this config file, but it doesn't look like the model is trained. What am I doing wrong?

**Comment by OhadRubin** [comment]
Can you provide more details? 
Is the loss decreasing? 
Did you make any changes to the code?
Is the metric `final_beam_acc` close to ~1? (This measures whether teacher forcing was successful, if it isn't ~1, it might be a due to a change you made to smbop.py).
